px-data-grid-navigation.html

```diff
@@ -8,24 +8,24 @@
   <template>
     <style>
       :host {
-        position: relative;
-        display: block;
+        display: flex;
+        align-items: center;
+        justify-content: left;
         color: var(--px-data-grid-navigation-text-color, #000);
-        padding-top: var(--px-data-grid-navigation-padding-top, 2px);
-        padding-bottom: var(--px-data-grid-navigation-padding-bottom, 2px);
+        background-color: var(--px-data-grid-navigation-background-color, transparent);
+        padding-top: var(--px-data-grid-navigation-padding-top, 5px);
```

Comment:
Where is `--px-data-grid-navigation-padding-top` defined?

Where is `--px-data-grid-navigation-padding-bottom` defined?

Reply:
I just removed them in latest commit.  I don't think those are useful and they were not defined anywhere.